$MirOS: ports/net/centericq/patches/patch-src_hooks_irchook_cc,v 1.1 2005/04/22 23:44:48 tg Exp $
--- src/hooks/irchook.cc.orig	Mon Dec 20 00:53:40 2004
+++ src/hooks/irchook.cc	Tue Jul  5 15:14:01 2005
@@ -1260,7 +1260,7 @@ const char * const command, const char *
 	map<string, time_t>::iterator i = irhook.pingtime.find(up(nick));
 
 	if(i != irhook.pingtime.end()) {
-	    sprintf(buf, _("PING reply from the user: %d second(s)"), time(0)-i->second);
+	    sprintf(buf, _("PING reply from the user: %d second(s)"), (int)(time(0)-i->second));
 	    em.store(imnotification(imcontact(nick, irc), buf));
 	}
 
